discovery-provider/src/api/v1/models/playlist_library.py

```diff
@@ -42,7 +42,7 @@ def output(self, key, obj, **kwargs):
 playlist_library_folder = ns.model(
     "playlist_library_folder",
     {
-        "type": "folder",
+        "type": fields.FormattedString("folder"),
```

Comment:
Not sure if I follow why FormattedString is necessary here, isn't this just a string? (looking at https://flask-restplus.readthedocs.io/en/stable/api.html#flask_restplus.fields.FormattedString)

Maybe you could add a comment?

Reply:
wait yeah i actually don't know, i just copied the use of `FormattedString` from the playlist and explore playlist models in this file 🤔

Reply:
ok i think it's because with String you can't supply a source string ("folder" in this case)

Reply:
haha yeah